atlasdb-timelock-server/src/test/java/com/palantir/atlasdb/timelock/paxos/LeaderPingHealthCheckTest.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.timelock.paxos;
+
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableSet;
+import com.palantir.leader.PingableLeader;
+
+public class LeaderPingHealthCheckTest {
+
+    @Test
+    public void shouldBeHealthyIfExactlyOneNodePingedSuccessfully() throws Exception {
+        PingableLeader leader1 = getMockOfPingableLeaderWherePingReturns(true);
```

Comment:
nit: not usually a fan of boolean parameters, but this could be a time for _three_ of them (repeated code 32-35, 42-45, 52-55)